src/main/java/codeu/model/data/User.java

```diff
@@ -58,4 +61,15 @@ public String getPasswordHash() {
   public Instant getCreationTime() {
     return creation;
   }
+
+  /** Sets the bio of this User. */
+  public void setBio (String aboutMe) {
+    bio = aboutMe;
```

Comment:
Will this write the new bio through to datastore?

Reply:
If not, when/where is it updating it to datastore, I have the same doubt? :octocat:

Reply:
It won't yet! which is why in the doPost() method of ProfileServlet.java, I have this after calling user.setBio(bio):

        // updates UserStore to store the bio for next time program opens
        userStore.updateUser(user);

which calls writeThrough in PersistentDataStore.java which I modfied to get the current bio before creating the new User and adding that.

Reply:
Here's where the writeThrough() writes the bio in PersistentDataStore.java

  /** Write a User object to the Datastore service. */
  public void writeThrough(User user) {
    Entity userEntity = new Entity("chat-users", user.getId().toString());
    userEntity.setProperty("uuid", user.getId().toString());
    userEntity.setProperty("username", user.getName());
    userEntity.setProperty("password_hash", user.getPasswordHash());
    userEntity.setProperty("creation_time", user.getCreationTime().toString());
    userEntity.setProperty("bio", user.getBio());
    datastore.put(userEntity);
  }